localstack/services/dynamodb/server.py

```diff
@@ -95,28 +96,26 @@ def _log_listener(self, line, **_kwargs):
         LOG.info(line.rstrip())
 
 
-def create_dynamodb_server(port=None) -> DynamodbServer:
+def create_dynamodb_server(
+    port=None, db_path: Optional[str] = None, clean_db_path: bool = False
+) -> DynamodbServer:
     """
     Creates a dynamodb server from the LocalStack configuration.
     """
     port = port or get_free_tcp_port()
-    ddb_data_dir = f"{config.dirs.data}/dynamodb" if config.dirs.data else None
```

Comment:
Should we still leave this line in here, to set `db_path` in case `config.dirs.data` (DATA_DIR) is configured?